core/lib/Thelia/Form/CustomerUpdateForm.php

```diff
@@ -202,4 +220,24 @@ public function verifyEmailField($value, ExecutionContextInterface $context)
             $context->addViolation(Translator::getInstance()->trans("email confirmation is not the same as email field"));
         }
     }
+
+    /**
+     * @return string
+     */
+    private function getFormatBirthday()
+    {
+        if ($this->getRequest()->getSession() != null) {
```

Comment:
Whoooo, seems like a copy/paste ;)
Why didn't you do a service that handle this formatter ?